frontend/src/structures/adminMessageUpdate.ts

```diff
@@ -0,0 +1,18 @@
+export default class AdminMessageUpdate {
+
+    public id: number;
+    public Type: string;
+    public Message: string;
+    public Display: boolean;
```

Comment:
Can we change the field names to match the ones inside the backend? For consistency's sake